### DISCUSSION TITLE
Security Onion 2.4 sensor node installation says install had problem when finishes.

### DISCUSSION THREAD

**Discussion by AzhariCyber10** [Security-Onion-Solutions/securityonion#11076] [2.4]
When installing sensor node of SO 2.4 fails and says INSTALL HAD PROBLEM. I dont know the problem, what can i do?
![photo_5976523696661184444_y](https://github.com/Security-Onion-Solutions/securityonion/assets/142520900/aa88738b-42f3-4fd8-a304-b05e56b1eed8)

**Comment by TOoSmOotH** [comment]
Check out line 2 of the message.
